build-coordinator/src/main/java/org/jboss/pnc/coordinator/builder/DefaultBuildCoordinator.java

```diff
@@ -563,6 +563,36 @@ private void processBuildTask(BuildTask task) {
         }
     }
 
+    public void completeNoBuild(BuildTask buildTask, CompletionStatus completionStatus) {
+        int buildTaskId = buildTask.getId();
+        BuildCoordinationStatus coordinationStatus = BuildCoordinationStatus.SYSTEM_ERROR;
+        try {
+            if (CompletionStatus.NO_REBUILD_REQUIRED.equals(completionStatus)) {
+                updateBuildTaskStatus(buildTask, BuildCoordinationStatus.REJECTED_ALREADY_BUILT);
+//TODO cancel should be here enable in 2.0 as CANCELed is not failed build
```

Comment:
What needs to be done here? Do we have a JIRA for it?
If not, it would be good to create a JIRA and add the JIRA number here.